Document using volume mounts with Docker image (#2205)
influxdata/docs-v2: content/influxdb/v2.0/get-started.md

```
_To run InfluxDB in [detached mode](https://docs.docker.com/engine/reference/run/#detached-vs-foreground), include the `-d` flag in the `docker run` command._

### Persist data outside the InfluxDB container

> 1. Create a new directory in which to store your data, and change to that directory.
>    ```sh
>    mkdir my-influxdb-docker-data
>    cd my-influxdb-docker-data/
>    ```

Reviewer: ```suggestion
1. Create a new directory to store your data in and navigate into the directory.
   
   ```sh
   mkdir path/to/influxdb-docker-volume && cd $_
   ```
```

Author: `path/to/influxdb-docker-volume` sounds to me like something that should already exist... How about `path/to/influxdb-docker-data-volume`?

2. From within your new data directory,
   run the InfluxDB Docker container with the `-v` flag as in the example below.
   This will persist the data from `/root/.influxdb2/` _inside_ the container to `my-influxdb-docker-data/` on the host file system.
   ```sh
   docker run \
       --name influxdb \
       -p 8086:8086 \
       -v $PWD:/root/.influxdbv2 \
       quay.io/influxdb/influxdb:v2.0.4 && \

>    ```

Reviewer: ~~~suggestion
2. From within your new directory, run the InfluxDB Docker container with the `--volume` flag to
   persist data from `/root/.influxdb2/` _inside_ the container to the current working directory in
   the host file system.
   
   ```sh
   docker run \
       --name influxdb \
       -p 8086:8086 \
       --volume $PWD:/root/.influxdbv2 \
       quay.io/influxdb/influxdb:v2.0.4
   ```
~~~

```
{{% /note %}}

### Console into the InfluxDB container (optional)

To use the `influx` command line interface, console into the `influxdb` Docker container:

```bash
docker exec -it influxdb /bin/bash
```

> ### Persist data outside the InfluxDB container (optional)

Reviewer: I don’t know that’d I’d even say this is optional. Mounting a local directory should be the default behavior we suggest. If they don’t specify a local mount, they will lose all their data any time the container restarts.
